psimol/configs/atom_properties.yml

```
@@ -138,18 +150,20 @@ Si:
   atomic_number: 14
   valence: 4
   covalent_radius:
-    - 1.11 # single bond
-    - 1.04 # double bond
+    - 1.16
+    - 1.07
+    - 1.02
 
 P:
   name: Phosphorus
   symbol: P
   mass: 31.0
   atomic_number: 15
-  valence: 5
+  valence: 3
```

```suggestion
  valence: 5
```
Phosphorus is organically usually at V oxidation state (in ATP, DNA etc).